src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -239,11 +238,14 @@ private static void initializeAndUpgradeSchema(
             0 >= VersionNumber.parse("2.1").compareTo(version),
             "All Cassandra nodes in Reaper's backend storage must be running version 2.1+");
 
-    try (
-        SchemaMigrationLock schemaMigrationLock = new SchemaMigrationLock(version, session, config);
-        Database database = new Database(cassandra, config.getCassandraFactory().getKeyspace())) {
+    try (Database database = new Database(cassandra, config.getCassandraFactory().getKeyspace())) {
 
       int currentVersion = database.getVersion();
+      Preconditions.checkState(
+          currentVersion == 0 || currentVersion >= 15,
+          "You need to upgrade from Reaper 1.2.2 at least in order to run this version. "
+          + "Please upgrade to 1.2.2 before performing this upgrade.");
```

Comment:
`"Please upgrade to 1.2.2, or greater, before performing this upgrade."`

Reply:
done